### ISSUE TITLE
[FEATURE] Adding Readme for spring-boot-starter implementation of google gemini ai

### ISSUE THREAD

**Issue by Suhas-Koheda** [langchain4j/langchain4j#2134]
**Is your feature request related to a problem? Please describe.**
It becomes very hard for a complete beginner to read all the documentation especially for a person who needs to complete their project or something within a particular time 

**Describe the solution you'd like**
So to make it easy i think we should be adding the thing that there are spring boot starters and quarkus integrations available for us in each model page 

I am requesting/proposing this since i had problems when i first used Langchain4j since I was in the lower version 
I had many problems fixing this I fixed it only when i actively participated in the discussions of pull requests 
I think any person in a hurry can use it with just the dependency and we can also add the readme file to github or just mention the properties they can choose and what all options they have 

**Describe alternatives you've considered**
Once after adding the dependency code snippet in the model page we can add the parameters and types possible or else we can redirect them to github and add readme to each integration 

**Additional context**
I just wrote my code for the configuration of google gemini ai integration for pull request and thought it would be helpful for newbies or people using just for their projects but not really interested in exploring since i personally faced the problem myself :)

**Comment by langchain4j** [comment]
@Suhas-Koheda totally agree, would you like to help us with this? 🙏

**Comment by Suhas-Koheda** [comment]
Yeah i would like to but it will take a lot of time and PR 
Coz i have my exams rn! :) 
It's all the markdown files to be edited right? 
@langchain4j 

And how are you hosting the markdown files?

**Comment by langchain4j** [comment]
@Suhas-Koheda good luck with exams! :)

Yes, documentation is written in markdown, we use [Docusaurus](https://docusaurus.io/).
Documentation files are located [here](https://github.com/langchain4j/langchain4j/tree/main/docs/docs) and hosted here: https://docs.langchain4j.dev/
Once the documentation is updated, https://github.com/langchain4j/langchain4j/actions/workflows/docs.yaml can be run to publish new docu version on https://docs.langchain4j.dev/

We can start with adding "Spring Boot Starter" sections to every documentaiton page of an integration that has SB starter implemented. As you suggested, it should contain a maven dependency of the SB starter, supported properties and an example of using it.

**Comment by Suhas-Koheda** [comment]
Yeah! 
I'll be on it! 
Thank you! 
Happy Monday: :)

**Comment by langchain4j** [comment]
@Suhas-Koheda thanks a lot in advance! I've assigned you.
Happy Monday as well :)

**Comment by ddobrin** [comment]
@Suhas-Koheda - let's get your PR in the spring-boot repo updated and then we can build the doc for it

**Comment by Suhas-Koheda** [comment]
Yeah sure!

**Comment by ddobrin** [comment]
Hi @Suhas-Koheda 
could you please provide. an update whether you are working on this issue at this time?
Thank you

**Comment by Suhas-Koheda** [comment]
@ddobrin 
Yeah i was waiting for my spring boot starter to get completed so that i could work upon this 
Actually there is some confusion related to safety setting configuration that has to be cleared 
Can you pleas ehave a look 
Thank you!

**Comment by ddobrin** [comment]
@Suhas-Koheda 
added comments in that PR

For this documentation, can I suggest a documentation pattern for the starter settings similar to [this approach](https://docs.spring.io/spring-ai/reference/api/chat/vertexai-gemini-chat.html#_chat_properties)?
